packages/adapter-netlify/index.js

```diff
@@ -92,6 +93,19 @@ export default function ({ split = false, edge = edge_set_in_env_var } = {}) {
 			} else {
 				await generate_lambda_functions({ builder, split, publish });
 			}
+		},
+
+		supports: {
+			// reading from the filesystem only works in serverless functions
+			read: ({ route }) => {
+				if (edge) {
+					throw new Error(
+						`${name}: Cannot use \`read\` from \`$app/server\` in route \`${route.id}\` when using edge functions`
```

Comment:
should we bother mentioning the route ID here or is that just misleading? whether it's edge or not doesn't depend on the route currently

Reply:
that was my thinking, yeah — it doesn't matter what the route is